packages/backend/src/nest/storage/storage.service.ts

```diff
@@ -313,94 +309,88 @@ export class StorageService extends EventEmitter {
       this.logger.error('Error closing channels db', e)
     }
 
-    try {
-      await this.certificates?.close()
-    } catch (e) {
-      this.logger.error('Error closing certificates db', e)
-    }
+    // try {
+    //   await this.certificates?.close()
+    // } catch (e) {
+    //   this.logger.error('Error closing certificates db', e)
+    // }
+
     try {
       await this.certificatesRequests?.close()
     } catch (e) {
       this.logger.error('Error closing certificates db', e)
     }
+
     try {
       await this.communityMetadata?.close()
     } catch (e) {
       this.logger.error('Error closing community metadata db', e)
     }
+
     await this.__stopOrbitDb()
     await this.__stopIPFS()
   }
 
-  public async updatePeersList() {
-    const allUsers = this.getAllUsers()
-    const registeredUsers = this.getAllRegisteredUsers()
-    const peers = [...new Set(await getUsersAddresses(allUsers.concat(registeredUsers)))]
-    console.log('updatePeersList, peers count:', peers.length)
-    const community = await this.localDbService.get(LocalDBKeys.COMMUNITY)
-    this.emit(StorageEvents.UPDATE_PEERS_LIST, { communityId: community.id, peerList: peers })
-  }
-
-  public async loadAllCertificates() {
-    this.logger('Getting all certificates')
-    this.emit(StorageEvents.LOAD_CERTIFICATES, {
-      certificates: this.getAllEventLogEntries(this.certificates),
-    })
-  }
-
-  public async createDbForCertificates() {
-    this.logger('createDbForCertificates init')
-    this.certificates = await this.orbitDb.log<string>('certificates', {
-      replicate: false,
-      accessController: {
-        write: ['*'],
-      },
-    })
-    this.certificates.events.on('replicate.progress', async (_address, _hash, entry, _progress, _total) => {
```

Comment:
Do we need to hook up on `replicate.progress`? I'd heard it's not reliable anyway and we process data on other events anyway

Reply:
So I don't remember about it not being reliable but I think listening on 'replicated' requires changing code a bit because replicate.progress gave you the information about certain entry.
If we **don't have to** change it right now I would do it in next iteration because the change itself can introduce regressions.

Reply:
There was a need to move caching usernames anyway so I removed hooking up on `replicate.progress` in-the-process